reference/3.0/Microsoft.PowerShell.Management/New-ItemProperty.md

```diff
@@ -53,31 +61,65 @@ NoOfEmployees : 822
 
 This command adds a new registry entry, NoOfEmployees, to the MyCompany key of the HKLM:\Software hive.
 
-The first command uses the Path parameter to specify the path to the MyCompany registry key.
-It uses the Name parameter to specify a name for the entry and the Value parameter to specify its value.
+The first command uses the *Path* parameter to specify the path of the MyCompany registry key.
+It uses the *Name* parameter to specify a name for the entry and the *Value* parameter to specify its value.
 
-The second command uses the Get-ItemProperty cmdlet to see the new registry entry.
-### Example 2
-```
-PS C:\> get-item -path HKLM:\Software\MyCompany | new-Itemproperty -name NoOfLocations -value 3
+The second command uses the `Get-ItemProperty` cmdlet to see the new registry entry.
+
+### Example 2: Add a registry entry to a key
+
+```powershell
+Get-Item -Path "HKLM:\Software\MyCompany" | New-ItemProperty -Name NoOfLocations -Value 3
 ```
 
 This command adds a new registry entry to a registry key.
-To specify the key, it uses a pipeline operator (|) to send an object representing the key to the New-ItemProperty cmdlet.
+To specify the key, it uses a pipeline operator (|) to send an object that represents the key to `New-ItemProperty`.
+
+The first part of the command uses the `Get-Item` cmdlet to get the MyCompany registry key.
+The pipeline operator sends the results of the command to `New-ItemProperty`, which adds the new registry entry, NoOfLocations, and its value, 3, to the MyCompany key.
+
+This command works because the parameter-binding feature of Windows PowerShell associates the path of the `RegistryKey` object that `Get-Item` returns with the *LiteralPath* parameter of `New-ItemProperty`.
+For more information, see [about_Pipelines](../Microsoft.PowerShell.Core/About/about_pipelines.md).
+
+### Example 3: Create a MultiString value in the registry
+
+```powershell
+$newValue = New-ItemProperty -Path "HKLM:\SOFTWARE\ContosoCompany\" -Name 'HereString' -PropertyType MultiString -Value @"
+This is text which contains newlines
+It can also contain "quoted" strings
+"@
+$newValue.multistring
+```
+
+```output
+This is text which contains newlines
+It can also contain "quoted" strings
+```
 
-The first part of the command uses the Get-Item cmdlet to get the MyCompany registry key.
-The pipeline operator (|) sends the results of the command to the New-ItemProperty cmdlet, which adds the new registry entry, NoOfLocations, and its value, 3, to the MyCompany key.
+```powershell
+$newValue = New-ItemProperty -Path "HKLM:\SOFTWARE\ContosoCompany\" -Name 'MultiString' -PropertyType MultiString -Value ('a','b','c')
+$newValue.multistring[0]
+```
+
+```output
+a
+```
+
+These two examples show two ways to create `MultiString` values with `New-ItemProperty`.

```

Comment:
These three sentences should be one paragraph. The alternative is to split Example 3 into two examples. One using Here-String and Example 4 using an array. I think I prefer splitting this into two examples. Which every you choose, make the same change in all versions.